dev/sparktestsupport/modules.py

```diff
@@ -18,9 +18,15 @@
 from functools import total_ordering
 import itertools
 import re
+import os
 
 all_modules = []
 
+if os.environ.get("AMPLAB_JENKINS"):
+    hadoop_version = os.environ.get("AMPLAB_JENKINS_BUILD_PROFILE", "hadoop2.7")
+else:
+    hadoop_version = os.environ.get("HADOOP_PROFILE", "hadoop2.7")
+
```

Comment:
We should have a log message to show which profile we are printing; otherwise, it is hard for us to know which profile is activated.

Reply:
Yes. we already have these log message:
![image](https://user-images.githubusercontent.com/5399861/57261352-07399700-709a-11e9-9074-f6faf2097f7d.png)
https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/105106/consoleFull

Reply:
What I mean, we should have a log message to show which profile is being actually used. 

Just relying on the command parameter lists does not sound very reliable

Reply:
OK. updated to:
![image](https://user-images.githubusercontent.com/5399861/57506520-9d85eb00-732e-11e9-8a7e-05ef7319d359.png)